### DISCUSSION TITLE
Conditional background change

### DISCUSSION THREAD

**Discussion by dyatelok** [wezterm/wezterm#4580] [Q&A] [answered]
Hi, guys! I currently use wayland and have some problems with full-screen transparency. Appears it's by design and I can't do much about it.

Is there a way to change config in wezterm conditionally? For example to change my background if I'm in the fullscreen mode?

**Comment by nekowinston** [comment, marked as answer]
Someone recently asked a similar question about changing opacity on the fly. This should give you some pointers: https://github.com/wez/wezterm/discussions/4516#discussioncomment-7437825
